_posts/theme/2020-4-22-hanCode.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: theme
+title: hanCode
+category: theme
```

Comment:
How about use "fork" for category since it is based on existing Typora themes.

Like https://github.com/typora/typora-theme-gallery/pull/96/files for example

Reply:
Sorry that I didn't know the `fork` category before. Have changed to `fork` now.